python/pyspark/sql/dataframe.py

```diff
@@ -1514,7 +1514,7 @@ def cube(self, *cols):
     @since(1.3)
     def agg(self, *exprs):
         """ Aggregate on the entire :class:`DataFrame` without groups
-        (shorthand for ``df.groupBy.agg()``).
+        (shorthand for ``df.groupBy().agg()``).
```

Comment:
Thank you for your contribution. Is this the last one? Could you check the other examples in this file, please?

Reply:
cc @HyukjinKwon

Reply:
> Thank you for your contribution. Is this the last one? Could you check the other examples in this file, please?

My pleasure. This is the only instance in the current version branch.

Reply:
Thanks!